docs/sim/agent.rst

```diff
@@ -72,6 +72,7 @@ different max_episode_len for each agent.
 - `ActionSpaceType.TrajectoryWithTime`: `(Sequence[float], Sequence[float], Sequence[float], Sequence[float], Sequence[float])` continuous action space using trajectory as times, x coordinates, y coordinates, headings, and speeds to interpolate the vehicle along the trajectory.
 - `ActionSpaceType.MPC`: `(Sequence[float], Sequence[float], Sequence[float], Sequence[float])` continuous action space using trajectory as x coordinates, y coordinates, headings, and speeds to adaptively perform controls on the vehicle model in an attempt to match the given trajectory. 
 - `ActionSpaceType.TargetPose`: `Sequence[float, float, float, float]` continuous action space with a single vehicle x coordinate, y coordinate, heading, and time delta to reach the given pose.
+- `ActionSpaceType.TargetPoseFixedTime`: `Sequence[float, float, float]` same as `ActionSpaceType.TargetPose` but the time delta is fixed to `0.1s`.
```

Comment:
Perhaps include `ActionSpaceType.RelativeTargetPose`.

Reply:
I have removed `TargetPoseFixedTime` since it is satisfied by just requiring `0.1` through asserting on the action space.

Reply:
I added `RelativeTargetPose`'s description.